sickbeard/manual_search.py

```diff
@@ -136,11 +137,12 @@ def collectEpisodesFromSearchThread(show):
 
     # Finished Searches
     searchstatus = SEARCH_STATUS_FINISHED
-    for search_thread in sickbeard.search_queue.FORCED_SEARCH_HISTORY:
+    for search_thread in sickbeard.search_queue.MANUAL_SEARCH_HISTORY:
         if show and not str(search_thread.show.indexerid) == show:
             continue
 
-        if isinstance(search_thread, sickbeard.search_queue.ForcedSearchQueueItem):
+        if isinstance(search_thread, sickbeard.search_queue.ManualSearchQueueItem) or
```

Comment:
can be passed as a tuple: if isinstance(search_thread, (sickbeard.search_queue.ManualSearchQueueItem, sickbeard.search_queue.ForcedSearchQueueItem))